### PR TITLE
Add utils to __init__.py

### PR DESCRIPTION
**Pull request by JayDwee**
image
fix for this issue

### CODE REVIEW

File: discord/ext/test/__init__.py [hunk 1]
```diff
@@ -7,4 +7,5 @@
 
 from . import backend
 from .runner import *
+from .utils import *
```
Comment (CraftSpider) [inline discord/ext/test/__init__.py:10]: Maybe instead explicitly import only the methods, to reduce the chance of more accidental imports

Comment (JayDwee) [reply discord/ext/test/__init__.py:10]: idk which ones people use though, I can go through and import all the methods, or are there some you want not imported?

Comment (CraftSpider) [reply discord/ext/test/__init__.py:10]: I was thinking just explicitly import all the methods

**Comment by JayDwee** [comment]
should be ready

**Review by CraftSpider — APPROVED** [review]

**Pull request merged**